### PR TITLE
ADD: relevance + new NER apis: people, places, organizations

### PR DESCRIPTION
**Pull request by madisonmay**

### CODE REVIEW

File: tests/IndicoIoTest.php [hunk 1]
```diff
@@ -8,7 +8,6 @@
```
Comment (anneCarlson) [inline tests/IndicoIoTest.php]: Once again, we need batch tests.

Comment (madisonmay) [inline tests/IndicoIoTest.php]: :+1:

File: tests/IndicoIoTest.php [hunk 2]
```diff
@@ -187,6 +186,45 @@ public function testTwitterEngagement()
...
+    {
+        self::skipIfMissingCredentials();
+        $text = "president";
+        $queries = "president";
+        $result = IndicoIo::relevance($text, $queries);
+        $this->assertGreaterThan(0.5, $result);
+    }
+
     public function testIntersections()
     {
```
Comment (anneCarlson) [inline tests/IndicoIoTest.php]: We should have a intersections and multi test that doesn't take any apis as input and defaults to all valid for each client lib.

Comment (madisonmay) [inline tests/IndicoIoTest.php]: :+1:

**Comment by anneCarlson** [comment]
Do we have a test of intersections and/or multi that doesn't take an api list and tests to make sure it returns the right apis? We should have that in every client lib. I'm afraid of new stuff breaking those two. Otherwise looks good!

**Comment by madisonmay** [comment]
There are no default arguments for either in node, so have to explicitly add your list of APIs.  Going to add a ticket for adding in that logic but want to get this pushed out ASAP.

**Pull request merged**